.circleci/config.yml

```diff
@@ -27,6 +27,22 @@ jobs:
             sleep 5
             # Assert the 1000_genomes index has 3500 documents
             expr $(curl -s 'http://localhost:9200/1000_genomes/_search' | jq -r '.hits.total') = "3500"
+            curl -XDELETE localhost:9200/1000_genomes
+            docker-compose stop
+      - run:
+          name: Run the GCS Indexer
+          command: |
+            echo ${GOOGLE_SERVICE_KEY} | base64 --decode > ${HOME}/.config/gcloud/application_default_credentials.json
+            # Note: data-explorer_default has already been created
+            cd gcs
+            # Run Elasticsearch in the background with the indexer in the foreground to prevent blocking the main thread
+            docker-compose up -d elasticsearch
+            BILLING_PROJECT_ID=${GOOGLE_PROJECT_ID} docker-compose up --build indexer
+            # For some reason index isn't available right after indexer terminates, so sleep
+            sleep 5
+            # Assert the 1000_genomes index has 3500 documents
+            expr $(curl -s 'http://localhost:9200/1000_genomes/_search' | jq -r '.hits.total') = "2559"
+            curl -XDELETE localhost:9200/1000_genomes
```

Comment:
Can you add delete for BigQuery indexer also?

Reply:
https://github.com/DataBiosphere/data-explorer-indexers/pull/48/files/589083b6440e5addd28c6d90938d09127ec56840#diff-1d37e48f9ceff6d8030570cd36286a61R30